saas_app/.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: '3.4'
```

Comment:
I would replace the content of this file with the other file. Take a look here: https://github.com/cds-snc/scan-websites/tree/main/.devcontainer where we have a similar set up.